alerts/openshift-virtualization-operator/KubeVirtHyperconvergedClusterOperatorPodSchedulingMightBeImbalanced.md

```diff
@@ -0,0 +1,21 @@
+# KubeVirtHyperconvergedClusterOperatorPodSchedulingMightBeImbalanced 
+
+## Meaning 
+
+This alert fires when a node contains more than then the maximum configured 
+number of images. The problem happens because the list of images on a node
+is shortened to that configured number, 50 by default. 
+
+## Impact 
+
+Pod scheduling might be imbalanced across nodes. 
+
+## Mitigation 
+
+As a workaround, you can increase the image limit by editing the `
+KubeletConfig` object, or disable it altogether by and setting the value of `
```

Comment:
Change to: 

As a workaround, you can increase the image limit by editing the
`KubeletConfig` object or you can disable the image limit and set the 
value of `nodeStatusMaxImages` to `-1`.

Will the user know how to disable the image limit in the KubeletConfig object?

Reply:
at first, I thought about linking https://docs.openshift.com/container-platform/4.12/nodes/nodes/nodes-nodes-managing.html
but not sure if that's a good idea since we would need to manage the URL version

Reply:
Correct. There is no way to avoid the version number in the URL, at least with our current tooling. It would probably be enough just to mention the name of the parameter if it is a simple change. If not, don't worry. We assume that users do not expect full documentation of every problem in the runbooks. That's what the docs are for.

Reply:
well, the parameter is there: `nodeStatusMaxImages`
maybe add an example config?

Reply:
Now I understand! In that case, this might work:

As a workaround, you can edit the `nodeStatusMaxImages` parameter of the `KubeletConfig` object, either by increasing its value or by setting its value to `-1` to disable the image limit.

What do you think?

Reply:
Sounds good, updated